editor/utils/dom.js

```diff
@@ -56,6 +62,25 @@ export function isEdge( container, start = false ) {
 }
 
 /**
+ * Check whether there is a node above node (or node itself) that matches selector
+ *
+ * @param  {Element} node DOM Element
+ * @param  {Boolean} selector  The selector to match
+ * @return {Element}           A node if one is found matching the selector, otherwise null
+ */
+export function closest( node, selector ) {
```

Comment:
I'm sure this is implemented somewhere else in a node_module that we are probably importing.